.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # These owners will be the default owners for everything in the repo.
 # Unless a later match takes precedence, these users will be requested
 # for review when someone opens a pull request.
-* @leahmcguire @tovbinm @wsuchy @Jauntbox @gerashegalov @nicodv
+* @leahmcguire @tovbinm @wsuchy @mrunfeldt @crupley @nicodv @michaelweilsalesforce @WintersLu
```

Comment:
is `@wsuchy` onboard or can be removed?

Reply:
Indeed, you can remove me as I no longer work at Salesforce. 
BTW Could someone add me to the list of contributors?

Reply:
@wsuchy added.

Reply:
Aw, I miss you guys.